### DISCUSSION TITLE
License refund

### DISCUSSION THREAD

**Discussion by filipesantosind** [aclap-dev/video-downloadhelper#1035] [License & Purchase requests]
### What's your issue?

I intended to purchase the Firefox version, but I accidentally bought the Chrome version instead.

Please process a refund for the Chrome version.

### The 10 first digits (only 10) of your license number or email address

75403b8c77

### I confirm that I did not not paste my full license number

- [X] I confirm

**Comment by jcfrog** [comment]
Hi,
You have been refunded. Should be available in a few business days.
